install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -64,7 +64,7 @@ ALTER TABLE `PREFIX_product_shop` ADD `show_condition` TINYINT(1) NOT NULL DEFAU
 /* PHP:ps_1700_add_payment_preferences_tab(); */;
```

Comment:
Line 62, same as last_connection_date, can you add : 

ALTER TABLE `PREFIX_product` CHANGE `available_date` `available_date` DATE NULL DEFAULT NULL;
ALTER TABLE `PREFIX_product_shop` CHANGE `available_date` `available_date` DATE NULL DEFAULT NULL;

and report it on db_structure.sql lines : 1433 & 1478


![sql3](https://cloud.githubusercontent.com/assets/9074610/19469637/755439d6-951c-11e6-9f8b-02700f22d457.png)